NotificationApi/NotificationApi/Controllers/ParticipantEmailNotificationsController.cs

```diff
@@ -183,20 +177,15 @@ public async Task<IActionResult> SendParticipantWelcomeEmailAsync(NewUserWelcome
         public async Task<IActionResult> SendParticipantMultiDayHearingConfirmationForExistingUserEmailAsync(
             ExistingUserMultiDayHearingConfirmationRequest request)
         {
-            var useNewTemplates = _featureToggles.UsePostMay2023Template();
             var notificationType = request.RoleName switch
             {
-                RoleNames.Individual when useNewTemplates => NotificationType.ExistingUserLipConfirmationMultiDay,
-                RoleNames.Individual when !useNewTemplates => NotificationType.HearingConfirmationLipMultiDay,
-                RoleNames.Representative when useNewTemplates => NotificationType.ExistingUserRepresentativeConfirmationMultiDay,
-                RoleNames.Representative when !useNewTemplates => NotificationType.HearingConfirmationRepresentativeMultiDay,
+                RoleNames.Individual => NotificationType.ExistingUserLipConfirmationMultiDay,
+                RoleNames.Representative => NotificationType.ExistingUserRepresentativeConfirmationMultiDay,
                 RoleNames.JudicialOfficeHolder when !request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationJohMultiDay,
                 RoleNames.JudicialOfficeHolder when request.Username.IsJudiciaryUsername() => NotificationType
                     .HearingConfirmationEJudJohMultiDay,
-                RoleNames.Judge when !request.Username.IsJudiciaryUsername() => NotificationType
-                    .HearingConfirmationJudgeMultiDay,
-                RoleNames.Judge when request.Username.IsJudiciaryUsername() => NotificationType
+                RoleNames.Judge => NotificationType
```

Comment:
I don't think `HearingConfirmationJudgeMultiDay` is behind a feature toggle but rather `request.Username.IsJudiciaryUsername()`

Reply:
reinstated